test/unit/beacon_api_test.exs

```diff
@@ -0,0 +1,55 @@
+defmodule BeaconApiTest do
+  use ExUnit.Case
+  use Plug.Test
+  use Patch
+  alias BeaconApi.Router
+
+  @moduletag :beacon_api_case
+
+  @opts Router.init([])
+
+  test "get state SSZ HashTreeRoot by head" do
+    root = Fixtures.Random.root()
+
+    resp_body = %{
+      data: %{root: "0x" <> Base.encode16(root, case: :lower)},
+      finalized: false,
+      execution_optimistic: true
+    }
+
+    {:ok, encoded_resp_body_json} = Jason.encode(resp_body)
+
+    patch(
```

Comment:
instead of using patch here, maybe you could setup the test by storing something in the db and then querying it here?

Reply:
ok, I think it is done. Although,  I am not sure about the `setup` part  :thinking: